src/components/entity-highlighter/FoundEntities.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export const FoundEntities = ({foundEntities, text, deleteEntity}) => {
+  return (
+    foundEntities.length > 0 && (
```

Comment:
Got rid of the additional `this.state.selectionStart === this.state.selectionEnd` check, so the use can select a part of the text and see which entities apply to that selection.